dimagi/pages/static/style/pages/about.scss

```diff
@@ -2,115 +2,105 @@
 @import 'app/variables/colors';
 @import 'app/variables/sizes';
```

Comment:
hey it looks like there are whitespace changes here...can you please undo this? 2 spaces in css. also `about.scss` shouldn't be touched in this PR